students/rmart300/lesson09/donor.py

```diff
@@ -0,0 +1,18 @@
+
+
+class Donor(object):
+
+    def __init__(self, name):
+
+        try:
+            name.split()[1]
+        except IndexError as e:
```

Comment:
You don't need the `as e` part of the `except` clause here, since you're not referencing the `e` variable in the `except` block. This syntax is generally used for logging or printing specific errors in the except block when you catch the exception.